PowerShell/ScubaGear/baselines/defender.md

```diff
@@ -483,7 +483,8 @@ Notifications to inform users and help educate them on the proper use of sensiti
 - _Last modified:_ June 2023
 - _Note:_ The custom policy referenced here is the same policy
           configured in [MS.DEFENDER.4.1v1](#msdefender41v1).
-
+- _MITRE ATT&CK TTP Mapping:_
```

Comment:
JCDC missed this change in a few locations: [MS.AAD.3.4](https://github.com/cisagov/ScubaGear/blob/1291-update-iceburg-scbs-to-adjudicate-comments-made-by-jcdc/PowerShell/ScubaGear/baselines/aad.md#msaad34v1), [MS.AAD.6.1](https://github.com/cisagov/ScubaGear/blob/1291-update-iceburg-scbs-to-adjudicate-comments-made-by-jcdc/PowerShell/ScubaGear/baselines/aad.md#msaad61v1), [MS.EXO.5.1](https://github.com/cisagov/ScubaGear/blob/1291-update-iceburg-scbs-to-adjudicate-comments-made-by-jcdc/PowerShell/ScubaGear/baselines/exo.md#msexo51v1), and [MS.EXO.7.1](https://github.com/cisagov/ScubaGear/blob/1291-update-iceburg-scbs-to-adjudicate-comments-made-by-jcdc/PowerShell/ScubaGear/baselines/exo.md#msexo71v1). 

[MS.EXO.10.2](https://github.com/cisagov/ScubaGear/blob/1291-update-iceburg-scbs-to-adjudicate-comments-made-by-jcdc/PowerShell/ScubaGear/baselines/exo.md#msexo102v1) contains the links but not the _MITRE ATT&CK TTP Mapping:_ header.

![Screenshot (193)](https://github.com/user-attachments/assets/dfe55422-5c6b-486f-bdf8-3e19aef59a17)